system/system_monitor/src/cpu_monitor/cpu_monitor_base.cpp

```diff
@@ -113,10 +119,16 @@ void CPUMonitorBase::checkUsage(diagnostic_updater::DiagnosticStatusWrapper & st
   // Remember start time to measure elapsed time
   const auto t_start = SystemMonitorUtility::startMeasurement();
 
+  tier4_external_api_msgs::msg::CpuUsage cpu_usage;
+  tier4_external_api_msgs::msg::CpuStatus STATUS;
```

Comment:
This does not have to declare a variable. Use using declaration instead.
```
using CpuStatus = tier4_external_api_msgs::msg::CpuStatus;
cpu_usage.all.status = CpuStatus.STALE;
```

Reply:
Using declaration caused build error. Currently, I don't change this.

Reply:
What error happened?

Reply:
The error was removed by correct expression.
First, I wrote "CpuStatus.STALE"